Combobox.js

```diff
@@ -9,13 +9,12 @@ define([
 	"delite/HasDropDown",
 	"delite/keys",
 	"./list/List",
-	"./LinearLayout",
-	"./Button",
+	"./Combobox/ComboPopup",
```

Comment:
As we are already in the ComboBox directory, and for the sake of not duplicating naming information already conveyed, I'm not sure if we have to restate "Combo" before Popup.